indy_node/test/nym_txn/test_nym.py

```diff
@@ -11,7 +11,7 @@ def test_non_steward_cannot_create_trust_anchor(
     sdk_wallet_client = sdk_add_new_nym(looper, sdk_pool_handle, sdk_wallet_steward)
     with pytest.raises(RequestRejectedException) as e:
         sdk_add_new_nym(looper, sdk_pool_handle, sdk_wallet_client, role=TRUST_ANCHOR_STRING)
-    e.match('None role cannot')
+    e.match('There is no accepted constraint')
```

Comment:
It would be great to have more descriptive exception than `There is no accepted constraint` for this kind of cases.